app/views/budgets/index.html.erb

```diff
@@ -176,6 +176,10 @@
   </div>
 <% end %>
 
+<% if has_banners? && is_banner_on_bottom? %>
+  <%= render "shared/banner" %>
+<% end %>
```

Comment:
This block is repeated in many places. Would it be possible to reduce duplication by using maybe a partial or a helper, or by replacing the current logic with `content_for`? Probably a partial is the easiest way:
```erb
<%= render "shared/bottom_banners" %>
```
Then we could do something similar for the top banners.